vite.config.ts

```diff
@@ -93,7 +93,7 @@ export default defineConfig({
     port: 4000,
     proxy: {
       "/api": {
-        target: process.env.CARE_API ?? "https://careapi.ohc.network",
+        target: process.env.CARE_API ?? "https://careapi.khavinshankar.dev",
```

Comment:
I think this change was committed by mistake. Rest of the PR LGTM

Reply:
Sure, It was just for testing, so that Nihal doesn't have to deploy a separate backend. Will definitely remove it. I will resolve this conversation once it's tested and removed.